bugbash.md

```diff
@@ -79,7 +79,15 @@ All quickstarts and tutorials are in the [IoT Plug and Play documentation](https
 
 #### EmbeddedC and Azure RTOS
 
-You can also test the **Quickstart on Azure RTOS for STM32L475 device**, [download this zip file](https://github.com/ericmitt/IotPnp/AzureRTOS_GA_Bugbash.zip) and follow the instructions in the readme pdf file.
+You can also test the **Quickstart on Azure RTOS for hardware we support**, [download this zip file based on the hardware and IDE](https://microsoft-my.sharepoint.com/:f:/p/liydu/EjI0F-b7aLdNhkT6AGnlvh4B9xixZzjN77CVO1zWZSNp4Q?e=F6pbWn) and follow the instructions in the readme pdf file.
```

Comment:
Could you mention where to find the readme pdf file? In `docs` folder.

Reply:
updated